### ISSUE TITLE
Unexpected error when trying to Simulate or saving to a PES file type

### ISSUE THREAD

**Issue by iampen** [inkstitch/inkstitch#570]
All of a sudden, I get this error when trying to Simulate or saving to a PES file type.

---
Tried to launch: inkstitch\bin\inkstitch
Arguments: ['inkstitch\\bin\\inkstitch', '--extension=simulate', 'C:\\Users\\ZAENAA~1\\AppData\\Local\\Temp\\ink_ext_XXXXXX.svgWR4XA0']
Debugging information:

Traceback (most recent call last):
  File "inkstitch.py", line 35, in <module>
    extension = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
  File "C:\Program Files\Inkscape\lib\python2.7/subprocess.py", line 325, in __init__
    errread, errwrite)
  File "C:\Program Files\Inkscape\lib\python2.7/subprocess.py", line 575, in _execute_child
    startupinfo)
WindowsError: [Error 2] The system cannot find the file specified
---

**Comment by iampen** [comment]
Apparently, this is a common error based on the other threads I've read. But so far, still no solution.

I am using a Windows machine. I installed InkScape and InkStitch more than 3 weeks ago, and I have been able to successfully create embroidery files (PES) since then. I have done satin stitches, satin column, and stroke stitches successfully.

Only yesterday, when I edited an SVG file to update the previous PES file I already exported, I encountered this error when I tried to save the SVG file to PES again. Until now, I get this error everytime a simulation is expected (params, live preview, embroider, etc.).

**Comment by lexelby** [comment]
Try reinstalling Ink/Stitch.  It's weird that it was working fine and now it is not.  I'm wondering if maybe an antivirus program deleted inkstitch.exe?

**Comment by iampen** [comment]
Thanks for replying, @lexelby!

I just checked the extensions folder and realized that the inkstitch.exe is missing. I checked the anti-virus history and found out that it quarantined the inkstitch.exe.
<img width="563" alt="quarantined_real-protect" src="https://user-images.githubusercontent.com/10723493/68917060-917a5a80-07a3-11ea-80d2-4a150e98d2cb.png">

I'm not sure yet how to put inkstitch.exe in the white list so McAfee won't quarantine it again.
